Character.h

```diff
@@ -9,66 +9,70 @@
 
 struct Character
 {
-    Character(int hp, int armor_, int attackDamage_ );
-    virtual ~Character() { }
-    
+    Character(int hp, int armor_, int attackDamage_);
+    virtual ~Character() = default;
+
     /*
      a pure virtual getName function.
      derived class stores the name, not the base class.
      */
-    
     virtual const std::string& getName() = 0;
     virtual std::string getStats() = 0;
-    
-    virtual void attack( Character& other );
+
+    virtual void attack(Character& other);
     void defend();
-    
-    void help( Character& other );
-    
+
+    void help(Character& other);
+
     int takeDamage(int damage);
-    
+
     int getHP() const { return hitPoints; }
     int getArmorLevel() const { return armor; }
     int getAttackDamage() const { return attackDamage; }
     bool getIsDefending() const { return isDefending; }
-    
-    const std::vector<std::unique_ptr<Item>>& getHelpfulItems() const { return helpfulItems; }
-    const std::vector<std::unique_ptr<Item>>& getDefensiveItems() const { return defensiveItems; }
-    
-    void boostArmor( int amount )
+
+    const std::vector<std::unique_ptr<Item>>& getHelpfulItems() const
+    {
+        return helpfulItems;
+    }
+    const std::vector<std::unique_ptr<Item>>& getDefensiveItems() const
+    {
+        return defensiveItems;
+    }
+
+    void boostArmor(int amount)
     {
         armor += amount;
-        std::cout << getName() << "'s armor level has been boosted to " << armor << std::endl;
+        std::cout << getName() << "'s armor level has been boosted to " << armor
+                  << std::endl;
```

Comment:
Does your clang-format do this automatically?

Reply:
yes, I set up a maximum line width